INSTALL.rst

```diff
@@ -330,6 +330,13 @@ openPMD API
 """""""""""
 - 0.12.0+ (bare minimum) / 0.13.0+ (for streaming IO)
 - *Spack*: ``spack install openpmd-api``
+- For usage in PIConGPU, the openPMD API must have been built either with support for ADIOS2 or HDF5.
+  If building the openPMD API from source (see below for that), those dependencies must be built and installed first.
```

Comment:
I would slightly adjust it as
`When building the openPMD API from source (described below), these dependencies must be built and installed first.`